src/main/java/news/blog/com/service/impl/ArticleServiceImpl.java

```diff
@@ -19,14 +24,21 @@
     }
 
     @Override
-    public ArticleEntity getArticle(Long id)
+    public ArticleDto getArticleDto(Long id)
     {
-            return articleRepository.findById(id).get();
+        return conversionService.convert(articleRepository.findById(id).orElseThrow(()-> new RuntimeException("Article not found " + id)), ArticleDto.class);
```

Comment:
let's discuss Exception throwing this evening

Reply:
let's discuss